routes/cube_routes.js

```diff
@@ -2031,6 +2031,16 @@ function insertComment(comments, position, comment) {
   }
 }
 
+function getOwnerFromComment(comments, position, comment) {
+  if (position.length <= 0) {
+    return '';
+  } else if (position.length == 1) {
+    return comments[position[0]].owner;
+  } else {
+    return getOwnerFromComment(comments[position[0]].comments, position.slice(1), comment);
+  }
```

Comment:
Why use recursion here instead of a for loop? it'll just muddy up the call stack.

Reply:
Also, what is the comment argument doing here?